pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -327,17 +327,19 @@ func (a *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) error
 	}
 
 	// finally - create missing workloads
+	errs := []error{}
 	for rem, remWl := range group.remotes {
 		if remWl == nil {
 			clone := cloneForCreate(group.local)
 			err := group.remoteClients[rem].client.Create(ctx, clone)
 			if err != nil {
 				// just log the error for a single remote
 				log.V(2).Error(err, "creating remote object", "remote", rem)
+				errs = append(errs, err)
 			}
 		}
 	}
-	return nil
+	return errors.Join(errs...)
```

Comment:
This is more than just "Use a second worker in the workload test".

I think this change has a user-facing impact. Please describe what scenario this fixes in the PR description and name accordingly.

Finally, is this change of logic tested - is there a test that would fail if reverted?

Reply:
> This is probably more than just "Use a second worker in the workload test", because this is non-test code.
> 
> I think this change has a user-facing impact. Please describe what scenario this fixes in the PR description and name accordingly.

Title and description updated.

> Finally, is this change of logic tested - is there a test that would fail if reverted?

`"wl with reservation, creates remote workloads, worker2 fails"`